.travis.yml

```diff
@@ -6,7 +6,7 @@ scala:
 jdk:
     - oraclejdk8
 script:
-  - sbt ++$TRAVIS_SCALA_VERSION coverage test tut
+  - sbt ++$TRAVIS_SCALA_VERSION coverage test tut doc
```

Comment:
It kind of seems like you'd only want to generate docs for 2.12 (well, the most recent version) if possible, rather than for all versions.
I have no idea if that's possible in Travis.